README.md

```diff
@@ -2,7 +2,7 @@
 [![Crates.io](https://img.shields.io/crates/v/flight_tracking_erau_se300.svg)](https://crates.io/crates/flight_tracking_erau_se300) 
 ![build](https://shields.io/github/workflow/status/FlightTrackingERAU/FlightTracking/CI/develop) 
 ![docs.rs](https://img.shields.io/docsrs/flight_tracking_erau_se300)
- ![License](https://img.shields.io/github/license/FlightTrackingERAU/FlightTracking)

```

Comment:
So here I think if add square brackets around this and then do (docs.rs/XXX) after the `]` like the top one it should work